core/src/components/nav/nav.tsx

```diff
@@ -503,22 +477,20 @@ export class Nav implements NavOutlet {
     return index > 0 ? views[index - 1] : undefined;
   }
 
-  // _queueTrns() adds a navigation stack change to the queue and schedules it to run:
-  // 1. _nextTrns(): consumes the next transition in the queue
-  // 2. _viewInit(): initializes enteringView if required
-  // 3. _viewTest(): ensures canLeave/canEnter Returns `true`, so the operation can continue
-  // 4. _postViewInit(): add/remove the views from the navigation stack
-  // 5. _transitionInit(): initializes the visual transition if required and schedules it to run
-  // 6. _viewAttachToDOM(): attaches the enteringView to the DOM
-  // 7. _transitionStart(): called once the transition actually starts, it initializes the Animation underneath.
-  // 8. _transitionFinish(): called once the transition finishes
-  // 9. _cleanup(): syncs the navigation internal state with the DOM. For example it removes the pages from the DOM or hides/show them.
+  /**
+   * Adds a navigation stack change to the queue and schedules it to run.
+   *
+   * @param ti The instructions.
+   * @param done Done function.
```

Comment:
Same thing here -- the comment does not provide any additional information that I could not already deduce from the type names below. Can we either make this more descriptive or remove?

Reply:
Removed